pkg/steps/multi_stage_test.go

```diff
@@ -26,6 +26,9 @@ import (
 	"github.com/openshift/ci-tools/pkg/testhelper"
 )
 
+// the multiStageTestStep implements the subStepReporter interface
+var _ SubStepReporter = &multiStageTestStep{}
+
```

Comment:
Huh why do we need an unused var here?

Reply:
This verifies that the *multistageTestStep implements the `SubStepReporter`, otherwise the only indication that we don't do that anymore would be the data not appearing